build-tools/packages/build-cli/src/commands/modify/lockfile.ts

```diff
@@ -37,7 +37,7 @@ export default class UpdateDependencyInLockfileCommand extends BaseCommand<
 			description: "Name of the dependency (npm package) to update.",
 			required: true,
 		}),
-		version: Flags.string({
+		version: semverFlag({
 			description: "Semver range specifier to use when updating the dependency.",
```

Comment:
@alexvy86 While this seems to only be used as a version at the moment, the flag description implies that a _range_ is also valid, not just a version. Should I update the description? Or maybe revert this part for now?

Reply:
Good catch. Updating the description should be fine. I think I misused the term anyway when writing this, I think I meant a version _specifier_ (?), i.e. `^x.y.z` is probably fine, but I wasn't planning for `a.b.c-x.y.z`

Reply:
The caret and tilde versions are range specifiers, too - they're just shorthand. I'll leave this a string and update the description to clarify.